src/auth/auth.ts

```diff
@@ -0,0 +1,14 @@
+import type { NextAuthOptions } from 'next-auth';
+import GammaProvider from '@/auth/GammaProvider';
+
+export const authConfig: NextAuthOptions = {
+  providers: [
+    GammaProvider({
+      authorizationURL: 'http://localhost:8081/api/oauth/authorize',
```

Comment:
Shouldn't these be environment variables?

Reply:
You are correct!